builds/ci/mqtt.yaml

```diff
@@ -0,0 +1,25 @@
+trigger: none
```

Comment:
Not sure that this pipeline should be called mqtt... Do you expect we add more in this pipeline vs create a new one?

Reply:
Talked with Vadim and we were thinking of making this the mqtt e2e pipeline. So once we add some e2e tests, it will run with this

Reply:
Ok, lets leave it mqtt then